### ISSUE TITLE
Create a 'how to get started' sheet for putting in team kits.

### ISSUE THREAD

**Issue by Adimote** [srobo/tasks#55]
To make it easier for rookie teams to get started, we should create a doc on common useful things to get started, i.e. docs, forums, IDE.

Also it should mention teams@ and the forums for how to get help, 

Maybe also mention our social media.

**Comment by RealOrangeOne** [comment]
It's worth noting this document should _link_ to the resources rather than duplicating their information

**Comment by PeterJCLaw** [comment]
See also https://github.com/srobo/tasks/issues/698

**Comment by trickeydan** [comment]
If you want this physically in the kits for this year, I need it by 8th november please.

**Comment by PeterJCLaw** [comment]
Since we now have the https://studentrobotics.org/docs/robots_101/ section in the docs can we call this done? I realise that's not in the kits, however when considered in combination with the microgames (not new) and the challenges (also relatively new) I think we're in a much better position now than when this ticket was opened.

cc @raccube for context around https://github.com/srobo/docs/pull/527

**Comment by PeterJCLaw** [comment]
@Adimote do you think that the https://studentrobotics.org/docs/competitor_resources/pre_kickstart_activities (alongside the other https://studentrobotics.org/docs/robots_101/ docs) covers this now? Or do you still think we need something to put actually in the kits?

**Comment by WillB97** [comment]
No, the point of this is a physical sheet to be in the kit that guides them through everything they need to do (including all the parts they need to purchase) to get to a minimal viable robot.
